### PR TITLE
Skip status check

### PR DESCRIPTION
**Pull request by pbalogh-sa**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #72
| License         | Apache 2.0

### What's in this PR?
In the case of the whitelisted releases, the validator skips checking images against policy.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] Related Helm chart(s) updated (if needed)

### Todo
After merged, tag chart and image-validator.

### CODE REVIEW

File: pkg/anchore/client.go [hunk 1]
```diff
@@ -183,7 +183,7 @@ func getImageDigest(imageRef string, isCached bool) (string, error) {
 }
 
 //CheckImage checking Image with Anchore
-func CheckImage(image string, isCached bool) (v1alpha1.AuditImage, bool) {
+func CheckImage(image string, isCached bool, isWl bool) (v1alpha1.AuditImage, bool) {
```
Comment (bonifaido) [inline pkg/anchore/client.go:186]: I would use a full name here instead of an abbreviation.

Comment (pbalogh-sa) [reply pkg/anchore/client.go:186]: I'll fix it

File: internal/app/validate.go [hunk 2]
```diff
@@ -155,11 +156,17 @@ func checkImage(pod *v1.Pod,
...
 			})
 		} else {
-			message = fmt.Sprintf("Image passed policy check: %s", image)
-
-			logger.Warn("Image passed policy check", map[string]interface{}{
-				"image": image,
-			})
+			if isWl {
+				message = fmt.Sprintf("Whitelisted release: %s", image)
+				logger.Warn("Whitelistetd release", map[string]interface{}{
```
Comment (orymate) [inline internal/app/validate.go:161]: wouldn't use warn level here. this is normal operation, no need for extra attention from operator
```suggestion
				logger.Info("skipping policy enforcement for whitelisted release", map[string]interface{}{
```

File: internal/app/validate.go [hunk 3]
```diff
@@ -126,24 +127,24 @@ func checkImage(pod *v1.Pod,
...
 		isCached := false
-		logger.Info("Checking cache", map[string]interface{}{
+		logger.Debug("Checking cache", map[string]interface{}{
 			"PodName": pod.Name,
 		})
 
 		_, found := cache.Get(image)
 		if found {
-			logger.Info("Cache found", map[string]interface{}{
+			logger.Debug("Cache found", map[string]interface{}{
```
Comment (orymate) [inline internal/app/validate.go:136]:
```suggestion
			logger.Debug("not submitting for scan again within ttl", map[string]interface{}{
```

File: internal/app/validate.go [hunk 4]
```diff
@@ -126,24 +127,24 @@ func checkImage(pod *v1.Pod,
...
 				"image": image,
 			})
 			isCached = true
 		} else {
-			logger.Info("Cache miss", map[string]interface{}{
+			logger.Debug("Cache miss", map[string]interface{}{
```
Comment (orymate) [inline internal/app/validate.go:141]:
```suggestion
			logger.Debug("submitting for scan the first time within ttl", map[string]interface{}{
```

**Review by orymate — APPROVED** [review]

**Review by bonifaido — APPROVED** [review]

**Pull request merged**